elixir/apps/domain/lib/domain/auth/adapters/jumpcloud/api_client.ex

```diff
@@ -60,10 +60,11 @@ defmodule Domain.Auth.Adapters.JumpCloud.APIClient do
     client = fetch_workos_client()
 
     case WorkOS.DirectorySync.list_groups(client, list_groups_params) do
-      {:ok, %WorkOS.List{data: groups, list_metadata: %{"after" => nil}}} ->
+      {:ok, %WorkOS.List{data: groups, list_metadata: %{"after" => nil}}} when is_list(groups) ->
```

Comment:
Do we need a case for `{:ok, %WorkOS.List{data: not_a_list, ... }` or is this not a possibility?

Reply:
If it's not a possibility, do we need the `when` clause?

Reply:
Good catch, looks like this has a `list` type defined here:

https://github.com/firezone/firezone/blob/main/elixir/apps/domain/lib/domain/auth/adapter/openid_connect/directory_sync.ex#L228